### DISCUSSION TITLE
How to delete an item by just clicking on it?

### DISCUSSION THREAD

**Discussion by ivancucer** [orchidjs/tom-select#265] [Q&A]
Hi,

I have some trouble understanding one thing.

**How to delete an item by just clicking on it?**

Currently, I have to click to select it first and then press the delete button on the keyboard.
Yes, there is an option to add `x` to delete an item but my goal is to delete it by just clicking on the entire one.

<img src="https://user-images.githubusercontent.com/2349195/141645624-6eb5560c-4ab2-4e3d-808e-b2607997a994.gif" width="301">

Any ideas?

Thanks in advance ❤️

**Comment by oyejorge** [comment]
Add a 'click' event listener to items via [render templates](https://tom-select.js.org/docs/#renderitem) and use [removeItem](https://tom-select.js.org/docs/api/#removeitemvalueorhtmlelement-silent)

**Reply by arfathyahiya** [reply]
Can you pls provide an example on how to do this?

**Reply by ivancucer** [reply]
@oyejorge yeah, I have the same question. It'd be great if you could provide an example of how to do that.

**Reply by oyejorge** [reply]
How about getting it started on a jsFiddle?

**Reply by ivancucer** [reply]
That's a great idea @oyejorge Here you go https://jsfiddle.net/ivancucer/ydkpztre/

**Reply by oyejorge** [reply]
Alright, so the next step is to [add a 'click' event listener to items via render templates](https://github.com/orchidjs/tom-select/discussions/265#discussioncomment-1647511)

**Reply by ivancucer** [reply]
@oyejorge I have no idea how to do that :) I'm still in the process of learning Javascript.

**Reply by ivancucer** [reply]
@oyejorge any ideas?
